src/sunadaptcontroller/imexgus/CMakeLists.txt

```diff
@@ -22,7 +22,6 @@ sundials_add_library(sundials_sunadaptcontrollerimexgus
     PUBLIC sundials_core
   INCLUDE_SUBDIR
     sunadaptcontroller
-  OBJECT_LIB_ONLY
```

Comment:
They are only used by ARKODE right now, but is there any reason to not install them like other modules (since in theory they could be used without ARKODE)?

Reply:
This would not make the user have to link them into ARKODE.  It just makes the libraries available to link to. It would be just like how we build in the iterative linear solvers, some matrices, serial vector, and the newton and fixed point, but also provide the standalone libraries.

Reply:
Unlike the other objects I don't think the controllers are as useful to have installed for use as a separate library.